resources/authgear/templates/en/web/authflowv2/__password_strength_meter.html

```diff
@@ -1,16 +1,19 @@
 {{ define "authflowv2/__password_strength_meter.html" }}
 <meter
-  class="{{ $.Classname }} password-strength-meter w-full"
+  class="{{ $.Classname }} password-strength-meter"
   min="1"
   max="5"
+  data-controller="password-strength-meter"
   data-password-policy-target="currentMeter"
+  data-password-strength-meter-target="currentMeter"
+  data-action="password-strength-updated@window->password-strength-meter#display"
```

Comment:
The original `<meter>` automatically renders when `value` changes. We should keep this capability. So instead of requiring another component to dispatch a CustomEvent, set up a mutation observer on `value` and update the rendering in PasswordStrengthMeterController. See `mirrorbutton.ts` as an example.